.github/workflows/build_static.yml

```
@@ -30,13 +30,13 @@ jobs:
       - name: Prepare AMD
         if: ${{ matrix.arch == 'amd' }}
         run: |
-          echo "TARGET=x86_64-unknown-linux-musl" >> $GITHUB_ENV
+          echo "TARGET_TRIPLET=x86_64-unknown-linux-musl" >> $GITHUB_ENV
           echo "DEVSHELL=.#crossShell" >> $GITHUB_ENV
 
       - name: Prepare ARM
         if: ${{ matrix.arch == 'arm' }}
         run: |
-          echo "TARGET=aarch64-unknown-linux-musl" >> $GITHUB_ENV
+          echo "TARGET_TRIPLET=aarch64-unknown-linux-musl" >> $GITHUB_ENV
```

Do you really mean `aarch64` here?
```suggestion
          echo "TARGET_TRIPLET=arch64-unknown-linux-musl" >> $GITHUB_ENV
```

---

The first `a` is for `Arm` I think: https://en.wikipedia.org/wiki/AArch64